_data/security.yml

```diff
@@ -79,6 +79,14 @@ websites:
       software: Yes
       doc: https://www.digicert.com/two-factor-authentication.htm
 
+    - name: DMARC Analyzer
+      url: https://www.dmarcanalyzer.com/
+      img: dmarcanalyzer.png
+      tfa: Yes
+      software: Yes
+      hardware: No
```

Comment:
```suggestion
```
A non-existing tfa-method must not be added with `:no`, because that's the implicit setting.

Reply:
Fixed ;)